### PR TITLE
Always set opal_cache_line_size in the app

### PR DESCRIPTION
**Pull request by ggouaillardet**

### CODE REVIEW

**Comment by rhc54** [comment]
I'm not convinced that is the best thing to do, but I don't really care. However, I am not sure the second commit here is correct, especially in the schizo/ompi component, so please don't do that until some more thought can be given to it as you might break (for example) the ability of the pmix reference server to support an OMPI job (since the two OPAL_MCA_PREFIX values won't match).

File: orte/mca/ess/base/ess_base_fns.c [hunk 1]
```diff
@@ -43,6 +43,13 @@
 
 #include "orte/mca/ess/base/base.h"
 
+static void set_cache_line_size() {
+    char * env;
+    if (NULL != (env=getenv(OPAL_MCA_PREFIX"opal_cache_line_size"))) {
```
Comment (jsquyres) [inline orte/mca/ess/base/ess_base_fns.c]: Should that be `OPAL_MCA_PREFIX "_opal_cache_line_size"`?

File: orte/mca/schizo/ompi/schizo_ompi.c [hunk 2]
```diff
@@ -979,6 +980,10 @@ static int setup_fork(orte_job_t *jdata,
         opal_setenv("OMPI_MCA_orte_bound_at_launch", "1", true, &app->env);
     }
 
+    char cache_line_size[16];
+    snprintf(cache_line_size, 16, "%d", opal_cache_line_size);
+    opal_setenv("OMPI_MCA_opal_cache_line_size", cache_line_size, true, &app->env);
```
Comment (jsquyres) [inline orte/mca/schizo/ompi/schizo_ompi.c]: If you use `OPAL_MCA_PREFIX` to read the env variable, you should use it to set the variable, too.

**Comment by rhc54** [comment]
Looked at it a bit more, and we do indeed have a problem here that bears some thought. Please don't merge this right now until we can think thru how we operate with the PMIx ref server. The ```OPAL_MCA_PREFIX``` stuff was all well-and-good when everything ran within its own project - maybe not so good when two projects need to interact.

**Comment by jsquyres** [comment]
bot:ompi:retest

**Comment by bwbarrett** [comment]
bot:ompi:retest

Brian broke the 32bit builds...

**Comment by AboorvaDevarajan** [comment]
Can one of the admins verify this patch?

**Comment by awlauria** [comment]
Since orte is gone, should we just close this?

**Comment by jsquyres** [comment]
I think there are conflicts in ORTE files (because they're now gone from master), but there's a lot of non-ORTE stuff on this PR.

**Comment by awlauria** [comment]
It's 2 files worth of changes that I can see: `pmix_cray.c` and `info.c`. Unless those files just got moved somewhere else?

Also it looks like `pmix_cray.c` no longer exists on master.

**Comment by rhc54** [comment]
Correct - none of the files altered in this PR exist any more. Question is: does something like this need to go into the OPAL or OMPI area for when a process is direct launched?

**Comment by jsquyres** [comment]
My mistake; I just looked closer, and this PR looks 100% stale / no longer valid.

**Pull request closed without merging**